CHANGELOG.md

```diff
@@ -5,6 +5,43 @@ information on the list of deprecated flags and APIs please have a look at
 https://docs.docker.com/engine/deprecated/ where target removal dates can also
 be found.
 
+## 1.12.2
+
+### Runtime
+
+* Check for non-nil container after match [#26049](https://github.com/docker/docker/pull/26049)
```

Comment:
Generally, we try to describe what is being fixed (not how). Perhaps something like;

```Markdown
Fix a panic due to a race condition filtering `docker ps` 
```